models/Prosit/Prosit_2023_intensity_XL_CMS2/notes.yaml

```diff
@@ -0,0 +1,36 @@
+description: |
+  Find out more about this model <a href="https://github.com/kusterlab/prosit">here</a>.
+
+  If you use predictions generated by this model please cite the following paper.
+
+
```

Comment:
Provide some documentation about your model here. Have a look at the timsTOF model for a guideline.